src/blocks/skeleton/Skeleton.types.ts

```diff
@@ -1,66 +1,31 @@
-import { CSSProperties, ReactNode } from 'react';
+import { ReactNode } from 'react';
 
-import { BlocksColors, ResponsiveProp, SpaceType, ValueOf } from '../Blocks.types';
-import { FlattenSimpleInterpolation } from 'styled-components';
+import type { BlockWithoutStyleProp, ResponsiveProp, SpaceType, ValueOf } from '../Blocks.types';
+import type { FlattenSimpleInterpolation } from 'styled-components';
 
 export type SkeletonResponsiveProps = {
-  /* Sets display css property */
-  display?: ResponsiveProp<CSSProperties['display']>;
   /* Sets height css property */
   height?: ResponsiveProp<string>;
   /* Sets margin css property */
   margin?: ResponsiveProp<SpaceType>;
-  /* Sets max-height css property */
-  maxHeight?: ResponsiveProp<string>;
-  /* Sets min-height css property */
-  minHeight?: ResponsiveProp<string>;
-  /* Sets max-width css property */
-  maxWidth?: ResponsiveProp<string>;
-  /* Sets min-width css property */
-  minWidth?: ResponsiveProp<string>;
-  /* Sets padding css property */
-  padding?: ResponsiveProp<SpaceType>;
   /* Sets width css property */
   width?: ResponsiveProp<string>;
   /* Sets border radius css property */
   borderRadius?: ResponsiveProp<string>;
```

Comment:
The border radius prop should accept values in the form of css variables similar to margin prop. I have introduced css variables for borderRadius prop in the buttons PR. 

- So you can just create type Blocks.types.ts called RadiusType: type RadiusType=`s${number}`;
- Use it here as  borderRadius?: ResponsiveProp<RadiusType>


It should work fine when box PR is merged.